pkg_defs/ska3-core/meta.yaml

```diff
@@ -1,214 +1,267 @@
+      - m2w64-gmp ==6.1.0  # [win]
+      - m2w64-libwinpthread-git ==5.0.0.4634.697f757  # [win]
+      - markupsafe ==1.1.1
+      - matplotlib ==3.2.2
+      - matplotlib-base ==3.2.2
+      - mccabe ==0.6.1
+      - menuinst ==1.4.16  # [win]
+      - mistune ==0.8.4
+      - mkl ==2020.1  # [linux or win]
+      - mkl ==2020.1  # [linux or win]
+      - mkl-service ==2.3.0
+      - mkl_fft ==1.1.0
+      - mkl_random ==1.1.1
+      - mock ==4.0.2
+      - more-itertools ==8.4.0
+      - mpld3 ==0.3
+      - msys2-conda-epoch ==20160418  # [win]
+      - nb_conda ==2.2.1
+      - nb_conda_kernels ==2.2.3
+      - nbconvert ==5.6.1
+      - nbformat ==5.0.7
+      - ncurses ==6.2  # [linux or osx]
+      - networkx ==2.4
+      - nose ==1.3.7
+      - notebook ==6.0.3
+      - numba ==0.50.1
+      - numexpr ==2.7.1
+      - numpy ==1.18.5
+      - numpy-base ==1.18.5
+      - numpydoc ==1.1.0
+      - oauthlib ==3.1.0
+      - olefile ==0.46
+      - openssl ==1.1.1g
+      - packaging ==20.4
+      - pandas ==1.0.5
+      - pandoc ==2.10
+      - pandocfilters ==1.4.2
+      - paramiko ==2.7.1
+      - parso ==0.7.0
+      - patchelf ==0.11  # [linux]
+      - pbr ==5.4.5
+      - pcre ==8.44  # [linux or osx]
+      - pexpect ==4.8.0
+      - pickleshare ==0.7.5
+      - pillow ==7.2.0
+      - pip ==20.1.1
+      - pkginfo ==1.5.0.1
+      - pluggy ==0.13.1
+      - prometheus_client ==0.8.0
+      - prompt_toolkit ==3.0.5  # [linux or osx]
+      - prompt_toolkit ==3.0.5  # [linux or osx]
+      - psutil ==5.7.0
+      - ptyprocess ==0.6.0  # [linux or osx]
+      - py ==1.9.0
+      - py-lief ==0.10.1
+      - pycodestyle ==2.6.0
+      - pycosat ==0.6.3
+      - pycparser ==2.20
+      - pycrypto ==2.6.1
+      - pyflakes ==2.2.0
+      - pygments ==2.6.1
+      - pyjwt ==1.7.1
+      - pylint ==2.5.3
+      - pynacl ==1.4.0
+      - pyopenssl ==19.1.0
+      - pyparsing ==2.4.7
+      - pyqt ==5.9.2
+      - pyreadline ==2.1  # [win]
+      - pyrsistent ==0.16.0
+      - pysocks ==1.7.1
+      - pytables ==3.6.1
+      - pytest ==5.4.3
+      - python ==3.8.3
+      - python-dateutil ==2.8.1
+      - python-docx ==0.8.10
+      - python-libarchive-c ==2.9
+      - pytz ==2020.1
+      - pywin32 ==227  # [win]
+      - pywinpty ==0.5.7  # [win]
+      - pyyaml ==5.3.1
+      - pyzmq ==19.0.1
+      - qt ==5.9.7
+      - qtconsole ==4.7.5
+      - qtpy ==1.9.0
+      - readline ==8.0  # [linux or osx]
+      - requests ==2.24.0
+      - requests-oauthlib ==1.3.0
+      - requests-toolbelt ==0.9.1
+      - ripgrep ==11.0.2  # [linux or osx]
+      - rope ==0.17.0
+      - ruamel_yaml ==0.15.87
+      - runipy ==0.1.5  # [win]
+      - scikit-learn ==0.23.1
+      - scipy ==1.5.0
+      - send2trash ==1.5.0
+      - setuptools ==49.2.0
+      - setuptools-scm ==4.1.2
+      - setuptools_scm ==4.1.2
+      - sherpa ==4.12.1  # [linux or osx]
+      - sip ==4.19.8  # [osx]
+      - sip ==4.19.8  # [osx]
```

Comment:
Also doubled but perhaps non-impacting.

Reply:
Good catch, for example, this should have been

```
sip ==4.19.13  # [linux or win]
sip ==4.19.8  # [osx]
```

Similar with the other repeated lines.